.github/workflows/gradle_publish.yml

```diff
@@ -0,0 +1,104 @@
+        uses: actions/checkout@v3
+        # Does not work out of the box with a private repository
+        # see https://github.com/actions/checkout/issues/287
+        #with:
+        #  submodules: true
+      - name: Link Deploy Keys for Submodules
+        uses: webfactory/ssh-agent@v0.7.0
+        with:
+          # This allows to add multiple individual deploy keys for multiple private submodules
+          # for a guide, see https://github.com/marketplace/actions/webfactory-ssh-agent
+          ssh-private-key: |
+            ${{ secrets.SSH_DEPLOY_KEY_CAMERA_SERVICE }}
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Add sentry.properties
+        run: |
+          echo "defaults.project=android-app" > sentry.properties
+          echo "defaults.org=cyface" >> sentry.properties
+          echo "auth.token=${{ secrets.SENTRY_API_TOKEN }}" >> sentry.properties
+
+      - name: Add gradle.properties
+        run: |
+          # Use the repository's automatically set up token to publish to the registry
+          cp gradle.properties.template gradle.properties
+          echo "githubUser=${{ github.actor }}" >> gradle.properties
+          echo "githubToken=${{ secrets.GITHUB_TOKEN }}" >> gradle.properties
+          # Inject Cyface API URL
+          echo "cyface.api=${{ secrets.PUBLIC_API }}" >> gradle.properties
+          # Inject Google Maps API key
+          echo "google.maps_api_key=${{ secrets.GOOGLE_MAPS_KEY }}" >> gradle.properties
+
+      # As we previously published the app with version code up to 320, we need add this as offset
+      - name: Calculate versionCode
+        env:
+          GITHUB_RUN_NUMBER: ${{ github.run_number }}
+        run: |
+          echo "VERSION_CODE=$((GITHUB_RUN_NUMBER + 320))" >> $GITHUB_ENV
+
+      # VersionCode and versionName are required to publish the app to Play Store
+      - name: Set versionCode and versionName
+        uses: chkfung/android-version-actions@v1.1
+        with:
+          gradlePath: measuring-client/build.gradle
+          versionCode: ${{ env.VERSION_CODE }}
+          versionName: ${{ github.ref_name }}
+
+      - name: Build App
+        run: ./gradlew bundleRelease
+
+      - name: Sign App
+        id: sign_app
+        uses: r0adkll/sign-android-release@v1
+        with:
+          releaseDirectory: measuring-client/build/outputs/bundle/release
+          signingKeyBase64: ${{ secrets.SIGNING_KEY }}
+          alias: ${{ secrets.SIGNING_ALIAS }}
+          keyStorePassword: ${{ secrets.SIGNING_KEY_STORE_PASSWORD }}
+          keyPassword: ${{ secrets.SIGNING_KEY_PASSWORD }} # optional
+
+      # We currently upload the app manually to Play Store, but we could automate this using a
+      # Google Service Account and the Github Action `r0adkll/upload-google-play`.
```

Comment:
Right now I don't like the idea to automate the upload to Play Store Console, yet. Let's keep this simple step manual for now.

Reply:
Isn't this directly live after the upload? On the App Store you could upload automatically, since you'd have to publish the build from within the Web-Interface before anyone gets to see it.

Reply:
Could be true, yes. We'll check this out when we feel the need to automate this last step :)